project/Projects/Presentation/DSKit/Sources/CommonUI/Picker/OneDayPicker.swift

```diff
@@ -0,0 +1,300 @@
+    }
+    
+    private func setLayout() {
+        
+        let conerRadius = 16.0
+        
+        calendarInputView.backgroundColor = .white
+        calendarInputView.layer.cornerRadius = conerRadius
+        calendarInputView.layoutMargins = .init(top: 0, left: 27, bottom: 64 + conerRadius, right: 27)
+        
+        let label = IdleLabel(typography: .Heading3)
+        label.textString = "접수 마감일"
+        label.textAlignment = .center
+        
+        let stack = VStack(
+            [
+                label,
+                calendar
+            ],
+            spacing: 31,
+            alignment: .fill
+        )
+        
+        [
+            dragSpace,
+            stack
+        ].forEach {
+            $0.translatesAutoresizingMaskIntoConstraints = false
+            calendarInputView.addSubview($0)
+        }
+        
+        NSLayoutConstraint.activate([
+            
+            dragSpace.topAnchor.constraint(equalTo: calendarInputView.layoutMarginsGuide.topAnchor),
+            dragSpace.leftAnchor.constraint(equalTo: calendarInputView.leftAnchor),
+            dragSpace.rightAnchor.constraint(equalTo: calendarInputView.rightAnchor),
+            
+            calendar.heightAnchor.constraint(equalToConstant: 320),
+            
+            stack.topAnchor.constraint(equalTo: dragSpace.bottomAnchor),
+            stack.leftAnchor.constraint(equalTo: calendarInputView.layoutMarginsGuide.leftAnchor),
+            stack.rightAnchor.constraint(equalTo: calendarInputView.layoutMarginsGuide.rightAnchor),
+            stack.bottomAnchor.constraint(equalTo: calendarInputView.layoutMarginsGuide.bottomAnchor),
+        ])
+        
+        [
+            calendarInputView
+        ].forEach {
+            $0.translatesAutoresizingMaskIntoConstraints = false
+            view.addSubview($0)
+        }
+        
+        NSLayoutConstraint.activate([
+            calendarInputView.rightAnchor.constraint(equalTo: view.rightAnchor),
+            calendarInputView.leftAnchor.constraint(equalTo: view.leftAnchor),
+            calendarInputView.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: conerRadius)
+        ])
+    }
+    
+    private func setObservable() {
+        
+        
+    }
+    
+    private func configureCalendar() {
+        
+        calendar.delegate = self
+        calendar.dataSource = self
+            
+        // 주간 월간 선택
+        calendar.scope = .month
+        
+        // Locale설정
+        calendar.locale = Locale(identifier: "ko_KR")
+         
+        // 스크롤 여부
+        calendar.scrollEnabled = true
+        calendar.scrollDirection = .horizontal
+        
+        
+        // 헤더 설정
+        calendar.appearance.headerTitleFont = DSKitFontFamily.Pretendard.semiBold.font(size: 18)
+        calendar.appearance.headerTitleColor = DSKitAsset.Colors.gray900.color
+        
+        calendar.appearance.headerDateFormat = "YYYY년 MM월"
```

Comment:
```suggestion
        calendar.appearance.headerDateFormat = "yyyy년 MM월"
```

대부분의 경우 YYYY를 써도 의도대로 동작하겠지만, 특별한 경우 오동작을 합니다 :)

Reply:
넵 숙지하고 있겠습니다!